test/react/render-props/react-root-context3.js

```diff
@@ -0,0 +1,45 @@
+var React = require('React');
+// the JSX transform converts to React, so we need to add it back in
+this['React'] = React;
+
+var { Provider, Consumer } = React.createContext(null);
+// this is done otherwise the test fails
+this['_Consumer'] = Consumer;
```

Comment:
We can remove this and use the approach from https://github.com/facebook/prepack/pull/2102